### PR TITLE
Standardize 2D/3D -> 2d/3d in documentation comments.

### PR DESCRIPTION
**Pull request by bangerth**
Like #14774, but in doc comments.

### CODE REVIEW

File: include/deal.II/grid/reference_cell.h [hunk 1]
```diff
@@ -201,7 +201,7 @@ class ReferenceCell
    * Return a Gauss-type quadrature matching the given reference cell (QGauss,
    * QGaussSimplex, QGaussPyramid, QGaussWedge).
    *
-   * @param[in] n_points_1D The number of quadrature points in each direction
+   * @param[in] n_points_1d The number of quadrature points in each direction
```
Comment (masterleinad) [inline include/deal.II/grid/reference_cell.h:204]: You will then also need to replace it in the signature.

Comment (bangerth) [reply include/deal.II/grid/reference_cell.h:204]: Ah yes, good point. So fixed.

**Comment by masterleinad** [comment]
```
/home/runner/work/dealii/dealii/include/deal.II/grid/grid_generator.h:377: warning: image file plate_with_a_hole_3d.png is not found in IMAGE_PATH: assuming external image.
/home/runner/work/dealii/dealii/include/deal.II/grid/grid_generator.h:975: warning: image file subdivided_cylinder_3d.png is not found in IMAGE_PATH: assuming external image.
/home/runner/work/dealii/dealii/include/deal.II/grid/grid_generator.h:1391: warning: image file eccentric_hyper_shell_2d.png is not found in IMAGE_PATH: assuming external image.
/home/runner/work/dealii/dealii/include/deal.II/grid/grid_generator.h:1392: warning: image file eccentric_hyper_shell_3d.png is not found in IMAGE_PATH: assuming external image.
```

**Comment by masterleinad** [comment]
You will also need to resolve conflicts.

**Comment by bangerth** [comment]
Let's give this another spin then!

**Review by marcfehling — APPROVED** [review]

**Review by masterleinad — APPROVED** [review]

**Pull request merged**